packages/import-handler/src/luaScripts/updateMetrics.lua

```diff
@@ -0,0 +1,34 @@
+local key = tostring(KEYS[1]);
+local status = tostring(ARGV[1]);
+local timestamp = tonumber(ARGV[2]);
+
+-- increment the status counter
+redis.call('HINCRBY', key, status, 1);
+
+if (status == "imported" or status == "failed") then
+  -- get the current metrics
+  local bulk = redis.call('HGETALL', key);
+  -- get the total, imported and failed counters
+  local result = {}
+	local nextkey
+	for i, v in ipairs(bulk) do
+		if i % 2 == 1 then
+			nextkey = v
+		else
+			result[nextkey] = v
+		end
+	end
```

Comment:
Should probably install https://github.com/prettier/plugin-lua 😄 

Also: why is the Lua needed? Could this update not be done in TS?

Reply:
Thanks @Nezteb , `prettier` is always needed for every repo and the Lua script here is to guarantee the operation is atomic